### PR TITLE
Fix indent checking script on Travis

### LINKED ISSUE #73: Travis failed to catch an indentation issue
Indentation was incorrect in https://github.com/exercism/xc/pull/71 but `bin/verify-indent` indicated that there was no issue. When run locally `bin/verify-indent` _did_ indicate that `indent.sh` needed to be run.

### PR DESCRIPTION
**Pull request by ryanplusplus**
Closes #73

### CODE REVIEW

**Comment by ryanplusplus** [comment]
I'm thoroughly confused.

I followed the [instructions for running a Travis Docker instance locally](https://docs.travis-ci.com/user/common-build-problems/#Running-a-Container-Based-Docker-Image-Locally) and this _does_ work there.

Additionally, this _did_ work before (see https://travis-ci.org/exercism/xc/builds/157757693). Comparing the previous failure and a recent non-failure there appear to be no differences between the Travis image or the worker.

So... any ideas?

File: bin/verify-indent [hunk 1]
```diff
@@ -1,7 +1,9 @@
-#!/bin/sh
+#!/bin/bash
 
 sh indent.sh
 
+git diff --quiet
```
Comment (ryanplusplus) [inline bin/verify-indent]: Finally found a fix. I have no idea why repeating the command here makes it work. Any guesses?

**Comment by patricksjackson** [comment]
Wow, that's odd. When I switched the check to `git diff-index --quiet HEAD` it worked fine every time (without the repeated call to git diff).

As for why it doesn't work, I don't really know. It's not a race condition (using `sleep` instead of the extra `git diff` doesn't help, and it will fail even if the tree is dirty before verify-indent is run). It's not specific to those particular files either. And most confusingly the original version works sometimes but not all the time. Just call `bin/verify-indent` several times repeatedly and it'll work randomly.

So yea, I have no idea either.

**Comment by ryanplusplus** [comment]
@patricksjackson I tried your simpler solution, but it identifies a failure even when there are no changed files. Although I hate the current hacky solution, I'm not sure how to proceed toward a simpler solution, particularly when I don't know of a way to consistently reproduce the issue locally.

**Comment by ryanplusplus** [comment]
@patricksjackson @palisaide are you guys okay with this change?

**Comment by patricksjackson** [comment]
Well, I don't really like it, but it seems to work.

I say add a short comment in the script to explain why they're both there just in case some future contributor forgets to look at the history. After that I'm ok to merge it.

**Comment by ryanplusplus** [comment]
@patricksjackson I added a comment

**Comment by patricksjackson** [comment]
LGTM

**Comment by ryanplusplus** [comment]
Updated with latest master

**Pull request merged**